internal/server/passwordreset.go

```
@@ -25,7 +24,7 @@ func (a *API) RequestPasswordReset(c *gin.Context, r *api.PasswordResetRequest)
 		return nil, err
 	}
 
-	org := data.MustGetOrgFromContext(c)
+	org := access.GetRequestContext(c).Authenticated.Organization
```

This should fail if not set I think

---

It will panic on the next line if it's not set.